src/ptvsd/wrapper.py

```diff
@@ -1880,7 +1880,6 @@ def on_exceptionInfo(self, request, args):
             is_json=True)
 
         body = resp_args['body']
-        body['breakMode'] = self.exceptions_mgr.get_break_mode(body['exceptionId'])
         self.send_response(request, **body)
```

Comment:
I don't see a change to `on_setExceptionBreakpoints` in wrapper.py. Am I missing something?